tests/ci_scripts/run_premerge_gpu.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# Copyright (c) 2020-2022, NVIDIA CORPORATION. All rights reserved.
```

Comment:
nit: also init year should be time when it was added `2022`

if modified in the future, then we will update it as 2022-20XX